yarn.lock

```diff
@@ -553,6 +553,26 @@
     "@types/yargs" "^15.0.0"
     chalk "^3.0.0"
 
+"@nsfw-filter/gif-frames@1.0.2":
+  version "1.0.2"
+  resolved "https://registry.yarnpkg.com/@nsfw-filter/gif-frames/-/gif-frames-1.0.2.tgz#a00731e56a944c7cfc8e65f26f6f2a6945432ea6"
+  integrity sha512-XZrbJWEN8YfVla5i+PD4Wj51rRlJ8OgnXiPjjOt/OsrbsCR9GZRD4jr953oNWcwiRaoIcOCFWQNMQukO7Yb1dA==
+  dependencies:
+    "@nsfw-filter/save-pixels" "^2.3.4"
+    get-pixels-frame-info-update "3.3.2"
+    multi-integer-range "3.0.0"
+
+"@nsfw-filter/save-pixels@^2.3.4":
```

Comment:
I just changed a little bit this lib, https://github.com/nsfw-filter/save-pixels